types/node/index.d.ts

```diff
@@ -5592,6 +5608,31 @@ declare module "constants" {
     export var ALPN_ENABLED: number;
 }
 
+declare module "module" {
+    class Module implements NodeModule {
+        static runMain(): void;
+        static wrap(code: string): string;
+        static _nodeModulePaths(path: string): string[];
+        static _load(request: string, parent?: NodeModule, isMain?: boolean): any;
```

Comment:
As far as I know non documented, internal APIs are not added to typings, only public APIs.

Reply:
Seems you're right. Don't see many private APIs in other typings. I've removed them.